set.mm

```diff
@@ -634131,6 +634131,31 @@ deduction version (Contributed by metakunt, 22-May-2024.) $)
       VOKUPUDUEZVLVMUGVIVPADEAVPDEUGAVPUFDEAVPDEQURUSUQUTVAVBVCVDVE $.
   $}
 
+  ${
+    $d M j k $.  $d N j $.  $d ch j $.  $d et j $.  $d j k ph $.  $d j ta $.
+    $d j th $.  $d k ps $.
+    uzindd.1 $e |- ( j = M -> ( ps <-> ch ) ) $.
+    uzindd.2 $e |- ( j = k -> ( ps <-> th ) ) $.
+    uzindd.3 $e |- ( j = ( k + 1 ) -> ( ps <-> ta ) ) $.
+    uzindd.4 $e |- ( j = N -> ( ps <-> et ) ) $.
+    uzindd.5 $e |- ( ph -> ch ) $.
+    uzindd.6 $e |- ( ( ph /\ th /\ ( k e. ZZ /\ M <_ k ) ) -> ta ) $.
+    uzindd.7 $e |- ( ph -> M e. ZZ ) $.
+    uzindd.8 $e |- ( ph -> N e. ZZ ) $.
+    uzindd.9 $e |- ( ph -> M <_ N ) $.
+    $( Induction on the upper integers that start at M. The first four
```

Comment:
```suggestion
    $( Induction on the upper integers that start at ` M ` . The first four
```
syntax highlighting from uzind, might as well

(note: it's possible you might need to rewrap uzindd because of the seemingly arbitrary 1 vs 2 spaces after the period thing)

Reply:
Should be fixed now.